website/docs/install/install-with-script.md

```diff
@@ -6,34 +6,97 @@ sidebar_label: Quickstart
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import ClientStackPng from '@site/static/img/client-stack.png';
 
-Prysm is an implementation of the [Ethereum proof-of-stake consensus specification](https://github.com/ethereum/consensus-specs) [<a href='#footnote-1'>1</a>]. In this guide, you’ll use Prysm to run a full Ethereum node [<a href='#footnote-2'>2</a>] and optionally a validator node [<a href='#footnote-3'>3</a>]. This will let you stake 32 ETH using hardware that you manage [<a href='#footnote-4'>4</a>].
 
-This is a beginner-friendly guide. Familiarity with the command line is expected, but otherwise this guide makes no assumptions about your technical skills or prior knowledge. Beginners are encouraged to **follow the footnotes** throughout this guide - the footnotes provide context and links to learning resources [<a href='#footnote-5'>5</a>].
+Prysm is an implementation of the [Ethereum proof-of-stake consensus specification](https://github.com/ethereum/consensus-specs) <a class='footnote' href='#footnote-1'>[1]</a>. In this guide, you’ll use Prysm to run an Ethereum node <a class='footnote' href='#footnote-2'>[2]</a> and optionally a validator <a class='footnote' href='#footnote-3'>[3]</a>. This will let you stake 32 ETH using hardware that you manage <a class='footnote' href='#footnote-4'>[4]</a>.
+ 
+This is a beginner-friendly guide. Familiarity with the command line is expected, but otherwise this guide makes no assumptions about your technical skills or prior knowledge. Beginners are encouraged to **follow the footnotes** throughout this guide - the footnotes provide context and links to learning resources <a class='footnote' href='#footnote-5'>[5]</a>.
+
+## Step 1: Familiarize yourself with nodes and networks
+
+Ethereum and its terminology are rapidly evolving, so it's important to align on some terminology definitions before proceeding:
```

Comment:
I kept having to remind myself how nodes, layers, and networks relate. This is probably obvious for OGs but I do think we can reduce a lot of friction by providing concise definitions. Hypothesizing that newcomers will *really* appreciate this, and that OGs won't be annoyed by it. Recommend trying it, seeing if anyone complains.